DXRModules/DeusEx/Classes/DXRPasswords.uc

```diff
@@ -643,6 +643,7 @@ function bool UpdateGoal(DeusExGoal goal, string oldpassword, string newpassword
 
 function bool UpdateNote(DeusExNote note, string oldpassword, string newpassword)
 {
+    if( note.bUserNote ) return false;
```

Comment:
this was a pretty hilarious discovery, the player can add a note manually to get the passwords in it replaced